felix/bpf/proxy/syncer.go

```diff
@@ -864,14 +864,14 @@ func (s *Syncer) getSvcNATKeyLBSrcRange(svc k8sp.ServicePort) ([]nat.FrontendKey
 	keys := make([]nat.FrontendKeyInterface, 0, len(loadBalancerSourceRanges))
 
 	for _, src := range loadBalancerSourceRanges {
-		if strings.Contains(src, ":") {
+		if strings.ContainsRune(src.String(), ':') {
```

Comment:
Will `.To4() == nil` work in this case instead of converting to string and find `:`?